packages/devtools_app/pubspec.yaml

```diff
@@ -64,11 +64,11 @@ dev_dependencies:
   build_web_compilers: '>=2.6.2 <3.0.0'
   matcher: ^0.12.3
   mockito: ^4.0.0
-  test: ^1.0.0
+  test: any
```

Comment:
comment that this is pinned by Flutter so we don't care.

Reply:
done